documentation/src/main/resources/pages/ditto/basic-placeholders.md

```diff
@@ -11,16 +11,78 @@ Placeholders may be used at several places in Ditto where something should be re
+| `topic:subject`         | Ditto Protocol [topic subject](protocol-specification-topic.html#messages-criterion-actions) (for message commands)                                                                             |
+| `topic:action-subject`  | either Ditto Protocol [topic action](protocol-specification-topic.html#action-optional) or [topic subject](protocol-specification-topic.html#messages-criterion-actions) (for message commands) |
+### Resource Placeholder
+| Placeholder    | Description                                                                           |
+|----------------|---------------------------------------------------------------------------------------|
+| `resource:type`         | the type of the Ditto Protocol [path](protocol-specification.html#path) , one of: `"thing" "policy" "message" "connection"`                                                                     |
+| `resource:path`         | the affected resource's path being the Ditto Protocol [path](protocol-specification.html#path) in JsonPointer notation, e.g. `/` when a complete thing was created/modified/deleted             |
+### Request Placeholder
 | Placeholder    | Description                                                                           |
 |----------------|---------------------------------------------------------------------------------------|
 | `{%raw%}{{ request:subjectId }}{%endraw%}` | the first authenticated subjectId which sent the command / did the request   |
+### Time Placeholder
+| Placeholder    | Description                                                                           |
+|----------------|---------------------------------------------------------------------------------------|
 | `{%raw%}{{ time:now }}{%endraw%}` | the current timestamp in ISO-8601 format as string in UTC timezone                    | 
 | `{%raw%}{{ time:now_epoch_millis }}{%endraw%}` | the current timestamp in "milliseconds since epoch" formatted as string  |
+### JWT Placeholder
+| Placeholder    | Description                                                                           |
+|----------------|---------------------------------------------------------------------------------------|
+| `{%raw%}{{ jwt:<claim-pointer> }}{%endraw%}` | Any standard or custom claims in the body of the authenticated JWT in JsonPointer notation - e.g., `jwt:sub` for the JWT "subject" or `jwt:extra/roles` for (potentially multiple) "roles" extracted from a JsonObject in the JWT named `extra`.<br/>JWT claims being JSON arrays are expanded (each value of the array is added) and provided [functions](#function-expressions) are performed on each array element.  |
+
+
+### Scope: Entity creation / modification
+
+Whenever creating or modifying [things](basic-thing.html) or [policies](basic-policy.html), the following placeholders
+may be used:
+* [request placeholder](basic-placeholders.html#request-placeholder)
```

Comment:
when adding a link within the same file, you can use:
```
[request placeholder](#request-placeholder)
```

This affects a lot of places in this file ;)